lib/transport/selenium-webdriver/appium.js

```diff
@@ -0,0 +1,59 @@
+const {WebDriver} = require('selenium-webdriver');
+const {Executor} = require('selenium-webdriver/http');
+const SeleniumServer = require('./selenium');
+const http = require('selenium-webdriver/http');
+const {isObject} = require('../../utils');
+
+
+module.exports = class AppiumServer extends SeleniumServer {
+  static get ServiceBuilder() {
+    return require('./service-builders/appium.js');
+  }
```

Comment:
Why do we need a static get as well as object level get function for ServiceBuilder